ddl/ingest/engine_mgr.go

```diff
@@ -38,7 +37,7 @@ func (m *engineManager) init(memRoot MemRoot, diskRoot DiskRoot) {
 }
 
 // Register create a new engineInfo and register it to the engineManager.
-func (m *engineManager) Register(bc *BackendContext, job *model.Job, indexID int64) (*engineInfo, error) {
+func (m *engineManager) Register(bc *BackendContext, jobID, indexID int64, schemaName, tableName string) (*engineInfo, error) {
```

Comment:
Why should we change input parameters?

Reply:
Because when it's called later(in the next PR), there's no `job` information.